src/index.mako.html

```diff
@@ -217,6 +217,7 @@
   % if device == 'mobile':
         <div ga-offline-bt></div>
   % endif
+        <div ga-tilt3d ng-if="globals.dev3d"></div>
```

Comment:
The advantage of `ng-if` is that the directive is not initialized at all if the condition is false. (but I'm not opposed to change it)